hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/menu/DashboardMenu.java

```diff
@@ -177,34 +181,49 @@ private VerticalLayout buildLinksAndVersion() {
         return links;
     }
 
-    private static Resource getImage() {
+    private static Resource getImage(final boolean gravatar) {
+        if (!gravatar) {
+            return new ThemeResource("images/profile-pic-57px.jpg");
+        }
+
         return UserDetailsFormatter.getCurrentUserEmail().map(email -> (Resource) new GravatarResource(email))
                 .orElse(new ThemeResource("images/profile-pic-57px.jpg"));
 
     }
 
-    private static Component buildUserMenu() {
+    private Component buildUserMenu(final UiProperties uiProperties) {
         final MenuBar settings = new MenuBar();
         settings.addStyleName("user-menu");
         settings.setHtmlContentAllowed(true);
 
-        final MenuItem settingsItem = settings.addItem("", getImage(), null);
+        final MenuItem settingsItem = settings.addItem("", getImage(uiProperties.isGravatar()), null);
 
         final String formattedTenant = UserDetailsFormatter.formatCurrentTenant();
-        final String formattedUsername = UserDetailsFormatter.formatCurrentUsername();
-        String tenantAndUsernameHtml = "";
         if (!StringUtils.isEmpty(formattedTenant)) {
-            tenantAndUsernameHtml += formattedTenant + "<br>";
+            settingsItem.setText(formattedTenant);
+            settingsItem.setDescription(i18n.getMessage("menu.user.description",
+                    UserDetailsFormatter.getCurrentTenant(), UserDetailsFormatter.getCurrentUser().getUsername()));
```

Comment:
Call to getCurrentTenant needs also to be adapted UserDetailsFormatter.getCurrentTenant().orElse("")